### DISCUSSION TITLE
"The Helm storage is stored on the remote cluster" - How to transfer Helm storage to remote cluster

### DISCUSSION THREAD

**Discussion by om3171991** [fluxcd/flux2#1619] [Q&A]
Hey Team,

Want to install helmRelease on a remote cluster. Following (https://fluxcd.io/docs/components/helm/helmreleases/) for Remote Cluster deployment but not able to clearly understand  - "The Helm storage is stored on the remote cluster".

As per the understanding Helm storage (kind of k8s secret) of any chart gets created only when it gets installed/deployed. So how can we create/store/transfer the desired chart on a remote cluster in form of helm storage?

Current Setup: Helm chart is committed in GitHub and helmRelease is reading chart.spec to deploy chart in a current-context cluster.

Expectation: deployment should happen in kubeConfig.secretRef defined cluster.

cc : @kingdonb @hiddeco @alex-lx @stealthybox @stefanprodan

**Comment by hiddeco** [comment]
See the section at the end of that page for information on how to target a remote cluster: https://fluxcd.io/docs/components/helm/helmreleases/#remote-clusters--cluster-api

**Reply by om3171991** [reply]
@hiddeco - thanks for quick reply. I had already seen that section and query is how can we transfer helm storage to remote cluster. It will be really great if you could explain with example or share any article for same.

**Reply by hiddeco** [reply]
You can theoretically just copy over the secrets, I would however just let it install from zero, which generally should not cause any issues?

**Comment by alex-lx** [comment]
案发违法.......................................................................................If you no longer wish to receive these emails, click on the following link: unsubscribe
